rotkehlchen/db/drivers/gevent.py

```diff
@@ -261,7 +261,7 @@ def __init__(
             )
         else:
             self._conn = sqlcipher.connect(  # pylint: disable=no-member
-                database=path,
+                database=str(path),
```

Comment:
The other part of the failure is this one, which is ratther weird. Only for sqlcipher if a path gets here it fails with type error. sqlite.connect works fine with paths.